scripts/variantstore/wdl/extract/utils.py

```diff
@@ -0,0 +1,58 @@
+import time
+import google.api_core.exceptions
+from google.cloud import bigquery
+
+
+def execute_with_retry(client, label, sql):
+    """Run a BigQuery SQL string with a label.
+
+    Three retries with incremental backoff if BiqQuery returns a 503, any other error is re-raised.
```

Comment:
Also for BadRequest right (looking at the code below?)

Reply:
good catch — changed 👍🏻